Specs/NumberWithUnit/Chinese/CurrencyModel.json

```diff
@@ -333,5 +333,124 @@
         "End": 4
       }
     ]
+  },
+  {
+    "Input": "人民币50元6角三分",
+    "Results": [
+      {
+        "Text": "人民币50元6角三分",
+        "Start": 0,
+        "End": 9,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "CNY",
+          "unit": "Chinese yuan",
+          "value": "50.63"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "人民币 美元",
```

Comment:
Please add some spec examples in English that show the previous issue is now fixed too.